### PR TITLE
Bug 1330391 - SETA-Make load_preseed do nothing if JobPriority table empty

### PR DESCRIPTION
**Pull request by alisha17**
This PR is another option for already opened PR #2203 .
Here, load_preseed does nothing if JobPriority table is empty, so the load_preseed will just wait until analyze_failure has eventually run. 
Bugzilla link: https://bugzilla.mozilla.org/show_bug.cgi?id=1330391

### CODE REVIEW

**Comment by armenzg** [comment]
@edmorley this change seems good to me. What do you think?

Thanks Alisha!

File: treeherder/seta/preseed.py [hunk 1]
```diff
@@ -13,30 +13,32 @@
...
-
-        # Deal with the case where we have a new entry in preseed
-        if len(queryset) == 0:
-            create_new_entry(job)
-        else:
-            # We can have wildcards, so loop on all returned values in data
-            for jp in queryset:
-                process_job_priority(jp, job)
+
+    if JobPriority.objects.count() == 0:
```
Comment (edmorley) [inline treeherder/seta/preseed.py]: Let's make this an early return instead, so we can avoid the `else` and resultant indentation.
Do we think it might also be worth logging a "skipping preseed since ..." message too?

Comment (wlach) [inline treeherder/seta/preseed.py]: Sorry for the drive-by comment, but I would also use exists for this type of case, it's a little more concise:

```py
if not JobPriority.objects.exists():
   ...
```

https://docs.djangoproject.com/en/1.11/ref/models/querysets/#exists

Comment (edmorley) [inline treeherder/seta/preseed.py]: Yeah I almost suggested that but the docs implied that `.exists()` might evaluate the queryset and so not be as optimal as `.count()` which it described as doing a `SELECT COUNT(*)`.

However your comment made me curious and in fact it appears the optimiser does the right thing for `.exists()` in this case...

```
$ ./manage.py shell_plus --print-sql
...
>>> JobPriority.objects.count()
SELECT COUNT(*) AS `__count`
FROM `seta_jobpriority`

Execution time: 0.000573s [Database: default]

>>> JobPriority.objects.exists()
SELECT (1) AS `a`
FROM `seta_jobpriority`
LIMIT 1

Execution time: 0.000752s [Database: default]
```

**Comment by armenzg** [comment]
@edmorley r?

LGTM
